### PR TITLE
build: add package dependencies

### PR DESCRIPTION
**Pull request by jsstevenson**
Let me know if this is missing anything

### CODE REVIEW

**Comment by mcannon068nw** [comment]
@rbasu101 can you confirm for dash app dependencies

File: pyproject.toml [hunk 1]
```diff
@@ -35,7 +41,6 @@ docs = [
     "sphinx-copybutton==0.5.2",
     "sphinxext-opengraph==0.8.2",
     "furo==2023.3.27",
-    "gravis==0.1.0",
```
Comment (jsstevenson) [inline pyproject.toml:38]: this was an accidental inclusion in the template

File: pyproject.toml [hunk 2]
```diff
@@ -21,12 +21,18 @@ requires-python = ">=3.8"
 description = "Python wrapper for accessing an instance of DGIdb v5 database"
 license = {file = "LICENSE"}
 dependencies = [
+    "pandas~=2.2.1",
```
Comment (jsstevenson) [inline pyproject.toml:24]: pinning pandas >=2 just because version 2 introduces some API breaks

**Comment by rbasu101** [comment]
I checked the dependencies and tested it out and it looks good to me!

**Pull request merged**